docs/components/movement-sensor/_index.md

```diff
@@ -510,10 +510,23 @@ Supported by GPS models.
 
 **Returns:**
 
-- [(Dict[str, float])](https://docs.python.org/3/library/stdtypes.html#str): The accuracy and/or precision of the sensor, if supported.
-  Contents depend on sensor model.
-
-For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/components/movement_sensor/index.html#viam.components.movement_sensor.MovementSensor.get_accuracy).
+- (MovementSensor.Accuracy): The precision and reliability metrics of the movement sensor, which vary depending on model.
+  This type contains the following fields:
+  - `accuracy` [(map<string, float>)](https://docs.python.org/3/glossary.html#term-mapping): A mapping of specific measurement parameters to their accuracy values.
+    The keys are string identifiers for each measurement (for example, "Hdop", "Vdop"), and the values are their corresponding accuracy levels as floats.
+  - `position_hdop` [(optional float)](https://docs.python.org/3/library/functions.html#float): Horizontal Dilution of Precision (HDOP) value.
+    It indicates the level of accuracy of horizontal measurements.
+    Lower values represent higher precision.
+  - `position_vdop` [(optional float)](https://docs.python.org/3/library/functions.html#float): Vertical Dilution of Precision (VDOP) value.
+    Similar to HDOP, it denotes the accuracy level of vertical measurements.
```

Comment:
```suggestion
    Similar to HDOP, it denotes the precision level of vertical measurements.
```
Maybe?

Reply:
@susmitaSanyal Do you know for sure here?